packages/perseus-editor/src/components/__stories__/locked-figures-section.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import {View} from "@khanacademy/wonder-blocks-core";
+import {color, spacing} from "@khanacademy/wonder-blocks-tokens";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+
+import LockedFiguresSection from "../locked-figures-section";
+import {getDefaultFigureForType} from "../util";
+
+import type {Meta, StoryObj} from "@storybook/react";
+
+export default {
+    title: "Perseus Editor/Components/Locked Figures Section",
```

Comment:
The `Perseus Editor` part of this can't include spaces. I recently discovered that things were not grouped correctly because of the spaces we had in the title here.  I _think_ spaces later on are fine, but for top-level areas, it needs to be `PerseusEditor/...`